server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -112,7 +112,9 @@ for host in $hosts; do
             nincomingerrs=$nincomingerrs+1
         fi
         # remove from results
-        prefix_value=$(cat ".prefix/prefix.${x%%.tar.xz}")
+        if [ -s .prefix/prefix.${x%%.tar.xz} ]; then
+            prefix_value=$(cat ".prefix/prefix.${x%%.tar.xz}")
+        fi
         if [ -L $RESULTS/$host/${x%%.tar.xz} ]; then
```

Comment:
The change looks good, but the if/elif/fi (lines 118-132) contains code duplication (in fact, the code should be an exact duplicate, but it isn't: the rm -rf  of the whole directory on line 126 is probably wrong: it should just rm  a single file). Better to define a variable for the two situations and then use it,keeping the code the same.